CHANGES.md

```diff
@@ -4,6 +4,8 @@
 
 ### Fixes
 
+* Images with ascending y-values are tiled correctly.
```

Comment:
Please explain the problem, where it occurred, i.e. `xcube serve`, then the fix so that users can understand what changed and can see that their issue has been solved. Please add issue number too.